source/fundamentals/crud/write-operations/delete.txt

```diff
@@ -0,0 +1,115 @@
+=================
+Delete a Document
+=================
+
+.. default-domain:: mongodb
+
+Overview
+--------
+
+You can remove documents by passing a query filter in the
+:java-docs:`deleteOne() </apidocs/mongodb-driver-sync/com/mongodb/client/MongoCollection.html#deleteOne(org.bson.conversions.Bson)>`, 
+:java-docs:`deleteMany() </apidocs/mongodb-driver-sync/com/mongodb/client/MongoCollection.html#deleteMany(org.bson.conversions.Bson)>` 
+or :java-docs:`findOneAndDelete() </apidocs/mongodb-driver-sync/com/mongodb/client/MongoCollection.html#findOneAndDelete(org.bson.conversions.Bson)>` methods.
+
+The ``deleteOne()`` method deletes a single document. If the query
+filter matches more than one document, the method will remove the first
+occurrence of a match in the collection. 
+
+The ``deleteMany()`` method deletes all documents that match the query
+filter.
+
+The ``findOneAndDelete()`` method deletes the first occurrence of a
```

Comment:
I think this can be expanded upon?

Like ``deleteOne()``, ``findOneAndDelete()`` deletes the first occurrence of a matching document in a collection. However, the find and delete portions are *atomic*...